test/test_facebook.py

```diff
@@ -351,5 +356,78 @@ def test_get_user_permissions_nonexistant_user(self):
             facebook.GraphAPI(token).get_permissions(1)
 
 
+class TestAppSecretProof(FacebookTestCase):
+
+    proof = '4dad02ff1693df832f9c183fe400fc4f601360be06514acb4a73edb783eec345'  # noqa
```

Comment:
Since the same access token and secret are used throughout each of these tests, can we define them here as well?

Also, since they are constants, they should be ALL CAPS.